plugins/modules/vmware_cluster_dpm.py

```diff
@@ -0,0 +1,188 @@
+      - Whether dpm should be automated or manual
+      type: str
+      default: automated
+      choices: automated, manual
+    host_power_action_rate:
+      description:
+      - specify host power action rate
+      type: int
+      default: 3
+      choices: [1, 2, 3, 4, 5]
+extends_documentation_fragment:
+- community.vmware.vmware.documentation
+
+'''
+
+EXAMPLES = r'''
+- name: Enable DPM
+  community.vmware.vmware_cluster_dpm:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datacenter_name: datacenter
+    cluster_name: cluster
+    enable_dpm: true
+    default_dpm_behaviour: automated
+    host_power_action_rate: 2
+  delegate_to: localhost
+
+'''
+
+RETURN = r'''
+'''
+
+try:
+    from pyVmomi import vim, vmodl
+except ImportError:
+    pass
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.vmware.plugins.module_utils.vmware import (
+    PyVmomi,
+    TaskError,
+    find_datacenter_by_name,
+    vmware_argument_spec,
+    wait_for_task,
+    option_diff,
+)
+from ansible.module_utils._text import to_native
+
+
+class VMwareCluster(PyVmomi):
+    def __init__(self, module):
+        super(VMwareCluster, self).__init__(module)
+        self.cluster_name = module.params['cluster_name']
+        self.datacenter_name = module.params['datacenter']
+        self.enable_dpm = module.params['enable_dpm']
+        self.default_dpm_behaviour = module.params['default_dpm_behaviour']
+        self.host_power_action_rate = module.params['host_power_action_rate']
+        self.datacenter = None
+        self.cluster = None
+
+        self.datacenter = find_datacenter_by_name(self.content, self.datacenter_name)
+        if self.datacenter is None:
+            self.module.fail_json(msg="Datacenter %s does not exist." % self.datacenter_name)
+
+        self.cluster = self.find_cluster_by_name(cluster_name=self.cluster_name, datacenter_name=self.datacenter)
+        if self.cluster is None:
+            self.module.fail_json(msg="Cluster %s does not exist." % self.cluster_name)
+
+    def check_dpm_config_diff(self):
+        """
+        Check DRS configuration diff
+        Returns: True if there is diff, else False
+
+        """
+        dpm_config = self.cluster.configurationEx.dpmConfigInfo
+        change_message = None
+        changes = False
+
+        if dpm_config.enabled != self.enable_dpm:
+            change_message = 'DPM enabled status changes'
+            changes = True
+            return changes, change_message
+        elif self.enable_dpm:
+            if dpm_config.hostPowerActionRate != self.host_power_action_rate or dpm_config.defaultDpmBehavior != self.default_dpm_behaviour:
+                change_message = 'DPM Host Power Action Rate and/or default DPM behaviour change.'
+                changes = True
+                return changes, change_message
+
+        return changes,change_message
```

Comment:
```suggestion
        return changes, change_message
```

There are some other problems, but let's start with the linter errors. Could yoou please add a space after the comma?